src/dso.ts

```diff
@@ -23,18 +43,49 @@ export const dso = {
    * @param force set true, will drop table before create table
    */
   async sync(force: boolean = false): Promise<void> {
-    for (const model of _models) {
-      await sync(_client, model, force);
+    if (_configClientReturn["mysql"]) {
+      for (const model of _models) {
+        await sync(_client, model, force);
+      }
+    } else if (_configClientReturn["postgres"]) {
+      for (const model of _models) {
+        await sync(_clientPostgres, model, force);
+      }
+    } else if (_configClientReturn["sqlite"]) {
+      for (const model of _models) {
+        await sync(_clientSqlite, model, force);
+      }
     }
   },
 
   /**
-   * Database client
+   * MySQL Database client
    */
   get client(): Client {
     return _client;
   },
 
+  /**
+  * Postgres Database client
+  */
+  get clientPostgres(): PostgresClient {
+    return _clientPostgres;
+  },
+
+  /**
+  * Sqlite Database client
+  */
+  get clientSqlite(): SqliteClient {
+    return _clientSqlite;
+  },
+
+  /**
+  * Current driver client
+  */
+  get configClientReturn(): _ClientType {
+    return _configClientReturn;
+  },
+
   /**
    * all models
```

Comment:
Once we establish the correct client abstraction, the client used or exported by dso at this time will be DsoClient